content/en/synthetics/browser_tests/actions.md

```diff
@@ -93,9 +93,13 @@ Asserts that an email was sent and whether specific values (`string`, `number`,
 
 #### Test your UI with custom JavaScript
 
-Test a custom assertion using your own JavaScript scripts. By default the assertion is done on the active page. If you want your JavaScript function to leverage a specific page element, you can select it using **Target Element** and then refer to it as the `element` parameter in your function.
+Test a custom assertion on the active page using your own JavaScript scripts. JavaScript assertions support all the JavaScript built-in methods. You can use JavaScript libraries if these are exposed in global variables or if you wrap their loading in a promise.
```

Comment:
I'd just say something like `JavaScript assertions run in the context of the active page so they can access everything defined in there`. The `if you wrap their loading in a promise` is just confusing, I'd put `if you need to load external libraries, see following snippet` and it should either put the snippet or link to a guide / faq / etc. somewhere that contains it and other snippets.